maintenance_request_purchase/views/purchase_order_views.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2019 Creu Blanca
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record model="ir.ui.view" id="purchase_order_form">
+        <field
+            name="name"
+        >purchase.order.form (in maintenance_equipment_purchase)</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_form" />
+        <field name="arch" type="xml">
+            <div name="button_box" position="inside">
+                <field name="maintenance_request_ids" invisible="1" />
+                <button
+                    name="action_view_maintenance_request"
+                    class="oe_stat_button"
+                    icon="fa-wrench"
+                    type="object"
+                    groups="purchase.group_purchase_user"
+                    attrs="{'invisible': [('maintenance_request_ids', '=', [])]}"
+                >
+                    <field
+                        name="maintenance_requests_count"
+                        widget="statinfo"
+                        string="Maintenance"
+                    />
+                </button>
+            </div>
+        </field>
+    </record>
+    <record model="ir.ui.view" id="purchase_order_line_search">
+        <field
+            name="name"
+        >purchase.order.line.search (in maintenance_equipment_purchase)</field>
+        <field name="model">purchase.order.line</field>
+        <field name="inherit_id" ref="purchase.purchase_order_line_search" />
+        <field name="arch" type="xml">
+            <field name="order_id" position="after">
+                <field name="name" />
+            </field>
+        </field>
+    </record>
```

Comment:
@etobella during migration of this addon I've noticed this fragment of code and I'm not know why it was added, do you remember its purpose?

Reply:
It is something I did 7 years ago, but I don't remember why :cry: 
I would recommend to remove that, as it makes no sense IMO.

Reply:
Done! I've placed it in a standalone commit, so it could be cherry-picked